sdk/src/loom/ModestMaps/core/TileGrid.ls

```diff
@@ -36,12 +36,19 @@ package loom.modestmaps.core
     import loom2d.math.Rectangle;
     import loom2d.text.TextField;
     
+    class GridBounds {
+        public var minCol:Number;
+        public var minRow:Number;
+        public var maxCol:Number;
+        public var maxRow:Number;
+    }
+    
     public class TileGrid extends Sprite implements IAnimatable
     {       
         /** if we don't have a tile at currentZoom, onRender will look for tiles up to 5 levels out.
          *  set this to 0 if you only want the current zoom level's tiles
          *  WARNING: tiles will get scaled up A LOT for this, but maybe it beats blank tiles? */ 
-        public static var MaxParentSearch:int = 5;
+        public static var MaxParentSearch:int = 8;
```

Comment:
Otherwise I got a black background flash while zooming in quickly by pinching, since touch zoom can go over 5 levels very quickly. It was less noticeable when parent traversal was allowed up to 8 levels.